### ISSUE TITLE
com.hazelcast.internal.serialization.impl.compact.schema.CompactSchemaReplicationStressTest.testSchemaReplication [API-1526]

### ISSUE THREAD

**Issue by olukas** [hazelcast/hazelcast#22044]
_master_ (commit 111323fed8ce48bf807eeaad28dda838b4ff9346)

Failed on IbmJDK8: 
- https://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-IbmJDK8-nightly/67/testReport/com.hazelcast.internal.serialization.impl.compact.schema/CompactSchemaReplicationStressTest/testSchemaReplication_driverType_CLIENT_/
- https://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-IbmJDK8-nightly/67/testReport/com.hazelcast.internal.serialization.impl.compact.schema/CompactSchemaReplicationStressTest/testSchemaReplication_driverType_MEMBER_/

Failed in variants `[driverType:CLIENT]` and `[driverType:MEMBER]`.

<details><summary>Stacktrace:</summary>

```
org.junit.runners.model.TestTimedOutException: test timed out after 300000 milliseconds
	at java.lang.Object.wait(Native Method)
	at java.lang.Object.wait(Object.java:218)
	at java.lang.Thread.join(Thread.java:705)
	at com.hazelcast.internal.serialization.impl.compact.schema.CompactSchemaReplicationStressTest.testSchemaReplication(CompactSchemaReplicationStressTest.java:96)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:90)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:55)
	at java.lang.reflect.Method.invoke(Method.java:508)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.util.concurrent.FutureTask.run(FutureTask.java:277)
	at java.lang.Thread.run(Thread.java:825)
```

</details>

Standard output can be found here https://console.aws.amazon.com/s3/buckets/j-artifacts/Hazelcast-master-IbmJDK8-nightly/67/

**Comment by olukas** [comment]
Failed on OracleJDK8: https://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-OracleJDK8-nightly/911/testReport/com.hazelcast.internal.serialization.impl.compact.schema/CompactSchemaReplicationStressTest/testSchemaReplication_driverType_CLIENT_/

**Comment by olukas** [comment]
Failed on IbmJDK8: 
- https://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-IbmJDK8-nightly/68/testReport/com.hazelcast.internal.serialization.impl.compact.schema/CompactSchemaReplicationStressTest/testSchemaReplication_driverType_CLIENT_/
- https://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-IbmJDK8-nightly/68/testReport/com.hazelcast.internal.serialization.impl.compact.schema/CompactSchemaReplicationStressTest/testSchemaReplication_driverType_MEMBER_/

**Comment by olukas** [comment]
@degerhz what's the reason for removing `5.2 Backlog` milestone? (it should be in `5.2 Backlog` as default or in `5.2` if we already know it should be fixed in `5.2`)

**Comment by mdumandag** [comment]
I am moving this to the 5.2 milestone. The fix is quite easy, and the test was failing frequently
